src/routes/extras/components/Extras.js

```diff
@@ -61,20 +62,20 @@ class Extras extends React.Component<PropsType> {
 
     const feedbackItems = [
       new FeedbackVariant(t('feedback:contentOfCity', {city: cityTitle}), EXTRAS_FEEDBACK_TYPE),
-      ...extras.map(xtra => new FeedbackVariant(t('feedback:extra', {extra: xtra.title}), EXTRA_FEEDBACK_TYPE)),
+      ...extras.map(extra => new FeedbackVariant(t('feedback:extra', {extra: extra.title}), EXTRA_FEEDBACK_TYPE)),
       new FeedbackVariant(t('feedback:technicalTopics'), CATEGORIES_FEEDBACK_TYPE)
     ]
 
-    navigation.navigate('FeedbackModal', { isPositiveFeedback, feedbackItems })
+    navigation.navigate('FeedbackModal', {isPositiveFeedback, feedbackItems})
```

Comment:
Change your autolinting rule or whatever does this :D

Reply:
I adjusted it :D